srml/contracts/rpc/runtime-api/src/lib.rs

```diff
@@ -45,6 +45,18 @@ pub enum ContractExecResult {
 	Error,
 }
 
+pub type GetStorageResult = Result<Option<Vec<u8>>, GetStorageError>;
+
+/// The possible errors that can happen querying the storage of a contract.
+#[derive(Eq, PartialEq, Encode, Decode, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
+pub enum GetStorageError {
```

Comment:
Why not use the type from `srml` directly?

Reply:
As per offline discussion, this would require another primitives crate. I am not sure if adding another crate is justified at this point, since the only use case for the moment would be avoiding one type definition, convertation logic.

Reply:
Can you briefly say what was the offline reason of not using the type in the SRML?

Reply:
introducing the primitives, only for this is an overkill, I agree. If you had other stuff in the contracts module which need to be shared and sort of postponed creating a primitive, then maybe now would be a good time. 

> This is not the end of the world, but it would be weird that it would depend on the entire crate

Yeah, the reason was that I also assumed this dependency is _bad/weird_, but then I was like: why though? Like, _really_ why? The rpc crate of one crate is really meant for that crate and only that crate imo. Hence, maybe it is fine to actually to depend on it?

I also don't have a strong opinion here and don't want to block the PR, so merge whenever needed, this is not a change request. But I do like the discussion.